### DISCUSSION TITLE
MeshCentral Router Customization

### DISCUSSION THREAD

**Discussion by EagleDotNet** [Ylianst/MeshCentral#6150] [General]
I searched a lot for a way to customize the MeshCentral Router program through the “config.json” file, as is the case with customizing the MeshCentral Windows Agent. Also, my search for the configuration at this link did not succeed.

https://github.com/Ylianst/MeshCentral/blob/master/sample-config-advanced.json

**Comment by si458** [comment]
what are you trying to customise exactally in the meshcentral router?
we have some docs already for the customise of the meshagent like name, colour, service name, etc
https://ylianst.github.io/MeshCentral/meshcentral/customization/#agent-branding

p.s docs here too for meshcentralrouter and custom links etc for port forwarding from the meshcentral 
https://ylianst.github.io/MeshCentral/meshrouter/

**Reply by EagleDotNet** [reply]
Thank you very much for the quick response, this is actually what I was searching for for hours.

**Reply by si458** [reply]
no worries! the website has loads of docs, but it is abit all over the place!
if you fancy updating/fixing it have a crack here hehe https://github.com/Ylianst/MeshCentral/tree/master/docs/docs

**Comment by EagleDotNet** [comment]
I have one last question: How can I uninstall meshcentral router after installing it on Windows?

**Reply by si458** [reply]
you can uninstall using the command line `meshcentralrouter.exe -uninstall`
but amazingly we have an install button in the app, but no uninstall button!
can you open an issue on the other repo for this and ill look at adding one in
https://github.com/Ylianst/MeshCentralRouter
